vocabularies/Common.xml

```diff
@@ -1425,6 +1425,14 @@ If the request succeeds the response will contain a JSON body of `Content-Type:
         <Annotation Term="Core.Description" String="Base URL for WebSocket connections" />
       </Term>
 
+      <Term Name="ValidationFunction" Type="Common.QualifiedName" Nullable="false" AppliesTo="Parameter Action Function">
```

Comment:
`Common.QualifiedName` identifies a function. Do we also want to be able to identify a particular function overload?

Reply:
I don't get how you call this function at all. You need the qualified name for a bound function, but what would be its binding parameter? I assume an unbound function is sufficient, but then you would need the name of the function import to call it. Once this is clarified, we can look at the rules for overloading (https://docs.oasis-open.org/odata/odata-csdl-xml/v4.01/odata-csdl-xml-v4.01.html#sec_FunctionOverloads)

Also: for a single non-binding parameter, the overload should be clear from its type. For an operation, I wonder if the validation function uses the same sequence of parameters or if there is a correlation based on names? Depending on that, there may again be no freedom w.r.t. overloads.